rbi/core/thread.rbi

```diff
@@ -522,8 +522,8 @@ class Thread < Object
   # See
   # [`::pending_interrupt?`](https://docs.ruby-lang.org/en/2.7.0/Thread.html#method-c-pending_interrupt-3F)
   # for more information.
-  sig {params(args: T.untyped).returns(T::Boolean)}
-  def pending_interrupt?(*args); end
+  sig {params(error: T::Class[T.anything]).returns(T::Boolean)}
+  def pending_interrupt?(error = nil); end
```

Comment:
Shouldn't `error` be of `T.nilable(T::Class[T.anything])`?

Reply:
If you're doing an explicit `Thread#pending_interrupt?(nil)`, sure.

If you're omitting the argument entirely it should work as is.

Reply:
Ah didn't see the linked issue initially. The actual issue is not this one but the `self.pending_interrupt?` further down I forgot to update. Was a Cmd+F failure by myself as I didn't realise there were two of them. Will fix.